MRETestBed/Assets/Scenes/SynchronizationTest-localhost-3clients.unity

```diff
@@ -0,0 +1,964 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
why are you adding this file?

Reply:
I see some weird behaviors with 3 clients in some tests, and here I can also test the late joining of the 3rd client.